src/features/app/components/Navbar/UserOptions.test.js

```diff
@@ -0,0 +1,80 @@
+import { fireEvent, render } from 'testUtils';
+
+import { userData, authenticationHeaders } from 'testUtils/mocks/auth';
+import UserOptions from 'features/app/components/Navbar/UserOptions';
+
+const state = {
+  auth: {
+    user: userData,
+    session: authenticationHeaders,
+  },
+};
+
+describe('UserOptions', () => {
+  it('renders an avatar when the user is logged in', () => {
+    const { getByRole } = render(<UserOptions />, { state });
+
+    getByRole('img', { name: 'avatar' });
+  });
+
+  it('does not render an avatar when the user is not logged in', () => {
+    const { queryByTestId } = render(<UserOptions />);
+
+    expect(queryByTestId('avatar')).toBeNull();
```

Comment:
does this work if we try getting the element by role? so we don't have to use testid?

Reply:
`getByRole` gives a fail if it doesn't find the element but maybe `queryByRole`works